### PR TITLE
add el4104

### PR DESCRIPTION
**Pull request by frankbrossette**
https://github.com/frankbrossette/linuxcnc-ethercat/blob/master/src/lcec_el41x4.c
https://github.com/frankbrossette/linuxcnc-ethercat/blob/master/src/lcec_el41x4.h

### CODE REVIEW

File: src/lcec_conf.h [hunk 1]
```diff
@@ -29,7 +29,7 @@
 #define LCEC_CONF_STR_MAXLEN 48
 
 #define LCEC_CONF_SDO_COMPLETE_SUBIDX -1
-#define LCEC_CONF_GENERIC_MAX_SUBPINS 32
+#define LCEC_CONF_GENERIC_MAX_SUBPINS 48
```
Comment (sittner) [inline src/lcec_conf.h]: Is this in corelation to the 4104 stuff or should we better open a new request to increase the constant?

File: src/lcec_main.c [hunk 2]
```diff
@@ -122,6 +123,9 @@ static const lcec_typelist_t types[] = {
   { lcecSlaveTypeEL4122, LCEC_EL41x2_VID, LCEC_EL4122_PID, LCEC_EL41x2_PDOS, lcec_el41x2_init},
   { lcecSlaveTypeEL4132, LCEC_EL41x2_VID, LCEC_EL4132_PID, LCEC_EL41x2_PDOS, lcec_el41x2_init},
 
+  // analog out, 4ch, 16 bits
+  { lcecSlaveTypeEL4104, LCEC_EL41x4_VID, LCEC_EL4104_PID, LCEC_EL41x4_PDOS, lcec_el41x4_init},
```
Comment (sittner) [inline src/lcec_main.c]: The constant define for LCEC_EL4104_PID is missing.

File: src/lcec_conf.h (hunk 1 above)
Comment (frankbrossette) [inline src/lcec_conf.h]: It has nothing to do with the 4104, it is only for more generic subpins

File: src/lcec_main.c (hunk 2 above)
Comment (frankbrossette) [inline src/lcec_main.c]: LCEC_EL4104_PID is defined in lcec_el41x4.h

**Comment by sittner** [comment]
Any Update about this? The files are still missing.

**Comment by sittner** [comment]
OK, I've copied the code from your fork and modified it to met the new pin export function. I'll enjoy your feedback.

**Pull request closed without merging**